tests/test_simple.py

```diff
@@ -364,6 +364,51 @@ def test_single_image():
     p = subprocess.Popen( [piffify_exe, 'simple.yaml'] )
     p.communicate()
 
+    # repeat using the plotify executables
```

Comment:
This test function is really getting rather unwieldy.  Can we please break it at the point where you start testing all the plots.  i.e. the comment `# Test that we can make rho statistics`.  At that point, it really changes focus to testing plots. So from there onward (including this new material) should be a new function called `test_plots` or some such.  You can pull out whatever code you need above that into a helper function if necessary to avoid too much code duplication.